infra/src/main/java/com/redhat/service/bridge/infra/models/dto/ConnectorStatus.java

```diff
@@ -5,6 +5,7 @@
 public enum ConnectorStatus {
     ACCEPTED("accepted"),
     TOPIC_CREATED("topic_created"),
+    MANAGED_CONNECTOR_CREATED("managed_connector_created"),
```

Comment:
I'm not keen on the semantics of these events (IMO they indicate a _request_ for XYZ) but I've gone with the flow :smile

Reply:
Events name shouldn't imply what's going to happen next, but what's happened when the event has been fired